Resources/Prototypes/Entities/Objects/Shields/shields.yml

```diff
@@ -217,3 +217,91 @@
                 SheetGlass:
                   min: 5
                   max: 5
+
+- type: entity
+  name: energy shield
+  parent: BaseItem
+  id: EnergyShield
+  description: Exotic energy shield that reflects almost all laser beams. But only them.
+  components:
+    - type: EnergySword #A little unsure about this one, but it works fine
```

Comment:
yeah, it's kinda weird, what exactly doesn't wrong if you remove it?

Reply:
The shield will not be able to switch between on and off state. Between the item, and the weapon (shield).

Reply:
Please do not re-use components like this, I don't want asteroid bees again.

Reply:
Unfortunately, this is the only way to "turn on" and "turn off" the shield correctly (I have not found others). Do you want a separate line for the shield?

Reply:
This needs coding before the PR can be merged.

Reply:
Done. Mostly copy-pasted, but separated from the energy sword.